cfme/tests/cloud_infra_common/test_relationships.py

```diff
@@ -136,3 +143,19 @@ def test_cloud_provider_relationships(appliance, provider, setup_provider, relat
     provider_view.entities.summary("Relationships").click_at(relationship)
     relationship_view = appliance.browser.create_view(view, additional_context={'object': obj})
     assert relationship_view.is_displayed
+
+
+@pytest.mark.provider([AzureProvider, EC2Provider, OpenStackProvider, GCEProvider])
+def test_cloud_provider_refresh_relationship(provider, setup_provider):
+    provider_view = navigate_to(provider, 'Details')
```

Comment:
**Required**: Use https://github.com/ManageIQ/integration_tests/blob/dbb30bb98f67793eef841cdd6c74dda2616fcf52/cfme/common/provider.py#L588 instead of  navigation and option click